include/rppt_tensor_audio_augmentations.h

```diff
@@ -0,0 +1,219 @@
+
+/******************** resample ********************/
+
+// Resample audio buffer based on the target sample rate
+
+// *param[in] srcPtr source tensor memory
+// *param[in] srcDescPtr source tensor descriptor
+// *param[out] dstPtr destination tensor memory
+// *param[in] dstDescPtr destination tensor descriptor
+// *param[in] inRate Input sampling rate (1D tensor of size batchSize)
+// *param[in] outRate Output sampling rate (1D tensor of size batchSize)
+// *param[in] srcLengthTensor source audio buffer length (1D tensor of size batchSize)
+// *param[in] channelsTensor number of channels in audio buffer (1D tensor of size batchSize)
+// *param[in] quality resampling quality, where 0 is the lowest, and 100 is the highest
+// *param[in] rppHandle HIP-handle for "_gpu" variants and Host-handle for "_host" variants
+// *returns a  RppStatus enumeration.
+// *retval RPP_SUCCESS : successful completion
+// *retval RPP_ERROR : Error
+
+RppStatus rppt_resample_host(RppPtr_t srcPtr, RpptDescPtr srcDescPtr, RppPtr_t dstPtr, RpptDescPtr dstDescPtr, Rpp32f *inRateTensor, Rpp32f *outRateTensor, Rpp32s *srcLengthTensor, Rpp32s *channelsTensor, Rpp32f quality, rppHandle_t rppHandle);
+
+/******************** normalize_audio ********************/
+
+// Normalizes the input audio buffer by removing the mean and dividing by the standard deviation
+
+// *param[in] srcPtr source tensor memory
+// *param[in] srcDescPtr source tensor descriptor
+// *param[out] dstPtr destination tensor memory
+// *param[in] dstDescPtr destination tensor descriptor
+// *param[in] srcLengthTensor source audio buffer length (1D tensor of size batchSize)
+// *param[in] channelsTensor number of channels in audio buffer (1D tensor of size batchSize)
+// *param[in] axisMask axis along which normalization needs to be done
+// *param[in] mean mean value to be subtracted from input
+// *param[in] stdDev standard deviation value to scale the input
+// *param[in] scale scaling factor applied to output
+// *param[in] shift value to which the mean will map in the output
+// *param[in] epsilon  value that is added to the variance to avoid division by small number
+// *param[in] ddof Delta Degrees of Freedom for Bessel’s correction
+// *param[in] numOfDims number of dimensions of input
+// *param[in] rppHandle HIP-handle for "_gpu" variants and Host-handle for "_host" variants
+// *returns a  RppStatus enumeration.
+// *retval RPP_SUCCESS : successful completion
+// *retval RPP_ERROR : Error
+
+RppStatus rppt_normalize_audio_host(RppPtr_t srcPtr, RpptDescPtr srcDescPtr, RppPtr_t dstPtr, RpptDescPtr dstDescPtr, Rpp32s *srcLengthTensor, Rpp32s *channelsTensor, Rpp32s axisMask,
+                                    Rpp32f mean, Rpp32f stdDev, Rpp32f scale, Rpp32f shift, Rpp32f epsilon, Rpp32s ddof, Rpp32s numOfDims, rppHandle_t rppHandle);
```

Comment:
@snehaa8 
I have removed the usage of numOfDims for normalize since it is needed. Please make similar change in this PR
https://github.com/sampath1117/rpp/commits/audio_merge_dev

Reply:
Sure incorporated this change